network/gossip/libp2p/network.go

```diff
@@ -157,14 +165,22 @@ func (n *Network) Identity() (map[flow.Identifier]flow.Identity, error) {
 
 // Topology returns the identities of a uniform subset of nodes in protocol state using the topology provided earlier
 func (n *Network) Topology() (map[flow.Identifier]flow.Identity, error) {
```

Comment:
just an fyi: my PR (vishal/network_refactor) changes the method signature of this method to return `flow.Identitylist` instead of a map.

Reply:
Thanks for the heads up, fixed in https://github.com/onflow/flow-go/pull/83/commits/f6cf7b4ca0dfd0782e172a8baf77e983616254d5.